slsDetectorServers/jungfrauDetectorServer/slsDetectorFunctionList.c

```diff
@@ -468,6 +477,119 @@ int setDefaultDacs() {
     return ret;
 }
 
+int readConfigFile() {
+
+    if (initError == FAIL) {
+        return initError;
+    }
+
+    char fname[128];
+    if (getAbsPath(fname, 128, CONFIG_FILE) == FAIL) {
+        return FAIL;
+    }
+
+    // open config file
+    FILE *fd = fopen(fname, "r");
+    if (fd == NULL) {
+        sprintf(initErrorMessage,
+                "Could not open on-board detector server config file [%s].\n",
+                CONFIG_FILE);
+        initError = FAIL;
+        LOG(logERROR, ("%s\n\n", initErrorMessage));
+        return FAIL;
+    }
+
+    LOG(logINFOBLUE, ("Reading config file %s\n", CONFIG_FILE));
+
+    // Initialization
+    const size_t LZ = 256;
+    char line[LZ];
+    memset(line, 0, LZ);
+    char command[LZ];
+
+    // keep reading a line
+    while (fgets(line, LZ, fd)) {
```

Comment:
Can some of the reading logic be merged into a common function for all detectors? I'm asking since many now have their own config files...

Reply:
Yes, but i was pushing it off for later due to the urgency.  Ill try to put it in for now if its fast. Else have to make a different PR for it.

Reply:
We decided to postpone this to another PR. Every detector has different commands and there are a bit too many to do it in this PR.